### PR TITLE
bug 1707664 fixing machineset labels

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1707664

@sunzhaohua2, will you PTAL?

### CODE REVIEW

**Comment by sunzhaohua2** [comment]
Line 70 and 71 should be removed.

File: modules/machineset-creating.adoc [hunk 1]
```diff
@@ -49,18 +49,15 @@ metadata:
   generation: 1 <1>
   labels:
     machine.openshift.io/cluster-api-cluster:  <infrastructure_id> <2>
-    machine.openshift.io/cluster-api-machine-role: <machine_label> <3>
-    machine.openshift.io/cluster-api-machine-type: <machine_label> <3>
-  name: <cluster_name>-<machine_label>-<AWS-availability-zone> <3> <4>
+  name: <cluster_name>-<machine_label>-<AWS-availability-zone> <3> <3>
```
Comment (sunzhaohua2) [inline modules/machineset-creating.adoc]: two <3>, one is ok.

**Comment by kalexand-rh** [comment]
Fixed. @sunzhaohua2, do you see anything else to change?

**Comment by sunzhaohua2** [comment]
@kalexand-rh  Sorry,  it should be line 73 and 74 are deleted.

The metadata section resembles the following YAML:
```
metadata:
  labels:
    machine.openshift.io/cluster-api-cluster: <infrastructure_id>
  name: <cluster_name>-<new_machine_label>-<AWS-availability-zone>
  namespace: openshift-machine-api
```

**Comment by kalexand-rh** [comment]
@sunzhaohua2, will you please take another look?

**Comment by sunzhaohua2** [comment]
LGTM, thanks!

**Comment by kalexand-rh** [comment]
Thanks! Merging.

**Pull request merged**